ddclient

```diff
@@ -2299,10 +2284,8 @@ sub get_ip {
         # no need to parse $reply
```

Comment:
I think we should change this such that `ip=<addr>` is processed through `ipv4_match` (and `ipv6_match`) in order to validate that the hard coded IP address provided by the user is in fact valid... so where we read in the ip address above place it onto `$reply` and then go through the following code to set `$ip`.

Reply:
Agreed, though that's out of scope for this pull request. I'll send a follow-up PR.